.github/workflows/nix-linux-cu9.yml

```diff
@@ -36,24 +36,4 @@ jobs:
           name: hasktorch
           signingKey: '${{ secrets.CACHIX_SIGNING_KEY }}'
       - run: cachix use iohk
-      - run: nix-build shell.nix --arg cudaVersion 9
-  build-ci-cache:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: free disk space
-        run: |
-          sudo swapoff -a
-          sudo rm -f /swapfile
-          sudo apt -y purge ghc* cabal-install* php* || true
-          sudo apt autoremove -y || true
-          sudo apt autoclean -y || true
-          docker rmi $(docker image ls -aq)
-          df -h
```

Comment:
It is necessary when there are not enough disks.
Do you want to erase it?

Reply:
I think this was done by accident, I put it back :)